.circleci/config.yml

```diff
@@ -75,8 +77,8 @@ jobs:
               --build-arg ENVIRONMENT_NAME="<<parameters.environment-name>>" \
               --build-arg PROCESS_NAME="thc" \
               --build-arg WORKTRAY_URL="<<parameters.worktray-url>>" \
+              --build-arg TENANCY_URL="<<parameters.tenancy-url>>" \
               --build-arg DIVERSITY_FORM_URL="<<parameters.diversity-form-url>>" \
-              .
```

Comment:
Bad rebase. Sorry. I'm working directly on `development` to get this build working.